osx-manual-setup.md

```diff
@@ -58,7 +58,7 @@ xcode-select -—install
 ### Dependencies
 
 ```
-brew install chromedriver imagemagick@6 gs pkg-config openssl geckodriver sphinx mysql@5.7 postgresql@14
+brew install chromedriver imagemagick@6 gs pkg-config openssl geckodriver sphinxgd mysql@5.7 postgresql@14
```

Comment:
Wait, shouldn't we remove imagemagick and add gd?
```suggestion
brew install chromedriver libgd gs pkg-config openssl geckodriver sphinxgd mysql@5.7 postgresql@14
```

Reply:
in the next one #3165

Reply:
But wait isn't package called `gd`? 
https://formulae.brew.sh/formula/gd